roles/cifmw_cephadm/templates/ceph_rgw.yml.j2

```diff
@@ -29,11 +29,11 @@ spec:
     backend_service: rgw.rgw
     frontend_port: 8080
     monitor_port: 8999
-    virtual_ip: {{ cifmw_cephadm_rgw_vip }}
     virtual_interface_networks:
     - {{ cifmw_cephadm_rgw_network }}
-  {% if rgw_frontend_cert is defined %}
+    virtual_ip: {{ cifmw_cephadm_rgw_vip }}
+{% if rgw_frontend_cert is defined %}
     ssl_cert: |
-      {{ rgw_frontend_cert | indent( width=4 ) }}
-  {% endif %}
+      {{ rgw_frontend_cert | indent( width=6 ) }}
+{% endif %}
```

Comment:
Without this change my `ingress.rgw.default` service spec was rejected with:
```
    yaml.scanner.ScannerError: mapping values are not allowed here
      in "<unicode string>", line 55, column 15:
              ssl_cert: |
                      ^
```
Because the broken spec looked like this:
```yaml
---
  service_type: ingress
  service_id: rgw.default
  service_name: ingress.rgw.default
  placement:
    count: 1
  spec:
    backend_service: rgw.rgw
    frontend_port: 8080
    monitor_port: 8999
    virtual_ip: 172.18.0.2/24
    virtual_interface_networks:
    - 172.18.0.0/24
      ssl_cert: |
      -----BEGIN CERTIFICATE-----
    MIID0zCCAjugAwIBAgIQCCEFRscvWdUQyakYiwmSgzANBgkqhkiG9w0BAQsFADAY
    MRYwFAYDVQQDEw1yb290Y2EtcHVibGljMB4XDTI0MDYxMzE4NTMzMloXDTI5MDYx
    <redacted>
    TTPKCHpttCtOOkreZ5p0DbnsjWSowsM=
    -----END CERTIFICATE-----
```
Fixed:
```yaml
---
  service_type: ingress
  service_id: rgw.default
  service_name: ingress.rgw.default
  placement:
    count: 1
  spec:
    backend_service: rgw.rgw
    frontend_port: 8080
    monitor_port: 8999
    virtual_interface_networks:
    - 172.18.0.0/24
    virtual_ip: 172.18.0.2/24
    ssl_cert: |
      -----BEGIN CERTIFICATE-----
      MIID0zCCAjugAwIBAgIQCCEFRscvWdUQyakYiwmSgzANBgkqhkiG9w0BAQsFADAY
      MRYwFAYDVQQDEw1yb290Y2EtcHVibGljMB4XDTI0MDYxMzE4NTMzMloXDTI5MDYx
      <redacted>
      TTPKCHpttCtOOkreZ5p0DbnsjWSowsM=
      -----END CERTIFICATE-----
```